design/design-thinking.md

```diff
@@ -0,0 +1,130 @@
+## Design Thinking
+
+### User Personas
+
+Personas are fictional characters that represent the different users we expect to interact with FlowForge as a brand and/or platform. You can find samples of the user personas we have at FlowForge in our [Design Thinking Output](./design-thinking).
+
+At FlowForge, we have many personas that will evolve over time. To give you an insight into what we are working with currently, here is a sample of our 5 MVPs (Most Valuable Personas) to-date:
+
+<div class="grid grid-cols-2 gap-4">
+    <div class="rounded-md bg-gray-900 text-white p-4 border-2 border-black">
+        <div class="py-2 text-center border-b-2 border-red-400">
+            <img class="w-32 rounded-md border-2 border-white m-auto" src="../images/personas/harry.jpg">
+            <label class="block mt-2 font-bold">Harry, the Hobbyist</label>
```

Comment:
I'd rather not focus on Hobbyists at this time of the company. I'd like to remove this persona completely, can we?

Reply:
Hobbyists are already the largest portion of the user base for FlowForge, to not include them as an MVP would be an error in my opinion. Whilst they may not bring in the largest amount of money, we still need to respect their opinion and considerations when in some capacity.

Reply:
In the workshop we ran, we concluded that an important persona would be a crossover between Harry and Neil. Someone that would use FlowForge to run a flow or two for some of their devices, but also contributed to their own Node-RED flows - I am happy to create this persona as a standalone, and merge Neil/Harry together.

Reply:
I am happy to re-order them, as there is some unconscious assessment of priority when reading them, which whilst isn't intentional, is unavoidable for a reader of the content.